### DISCUSSION TITLE
Supported postgres and plv8 versions?

### DISCUSSION THREAD

**Discussion by youqingz** [JasperFx/marten#2071] [Q&A] [answered]
As title, what are the supported postgres and plv8 versions for v3 and v4?
I'm not able to find the information in the documentation.

**Comment by mysticmind** [comment, marked as answer]
Marten works with Postgres 9.6 or above across all Marten versions. You can use the relevant plv8 extension appropriately for the chosen Postgres version.

**Reply by mysticmind** [reply]
Also note that plv8 is an optional requirement.

**Reply by youqingz** [reply]
Thanks!
